extensions/gdx-setup/src/com/badlogic/gdx/setup/SetupPreferences.java

```diff
@@ -29,9 +29,30 @@
 import java.util.Map.Entry;
 import java.util.Properties;
 
-public class SetupPreferences {	
+public class SetupPreferences {
+
+	private static final File[] files = new File[] {
+		// $HOME/.gdxsetup
+		new File(new File(System.getProperty("user.home")), ".gdxsetup"),
+		// $CONFIG_HOME/gdxsetup/config
+		new File(new File(new File(findConfigHomePath()), "gdxsetup"), "config")};
+
+	private static final String findConfigHomePath () {
+		Map<String, String> env = System.getenv();
+		if (env.containsKey("XDG_CONFIG_HOME")) return env.get("XDG_CONFIG_HOME");
+		// TODO add windows and mac config home paths
```

Comment:
I'm not sure about this TODO; is this something you want to add to the PR or something that would be added after the PR by someone else? As I understand it, this step is optional anyway, and the code will work on Windows and Mac with the current preferences location.

Reply:
I know there is the env var APPDATA in windows, but I'm not a Windows user so I will leave it for any windows user that is as ** as me and cannot handle random files in their ~.